src/components/DiagramFrame/utils.ts

```diff
@@ -1,14 +1,24 @@
-import { IGitBranch, ILookmlModelExplore } from "@looker/sdk/lib/sdk/3.1/models"
+import { IGitBranch, ILookmlModel, ILookmlModelExplore } from "@looker/sdk/lib/sdk/3.1/models"
 import { SelectOptionProps, theme } from "@looker/components"
-import { OVERRIDE_KEY_SUBTLE } from "../../utils/constants"
+import { OVERRIDE_KEY_SUBTLE, DIAGRAM_IGNORED_MODELS } from "../../utils/constants"
+import { ExploreDropdown } from "./types"
 
+/**
+ * styles the view list according to visibility status
+ * @param disabled - whether or not the current view is disabled
+ */
 export function viewDisabled(disabled: boolean) {
   if (!disabled) {
     return theme.colors.text1
   }
   return undefined
 }
 
+/**
+ * prepares the Git Branch dropdown data
+ * @param gitBranch - currently selected branch obj
```

Comment:
Better comment but function itself could be simplified. No need to do as part of this PR.

Something like
```
export const getBranchOptions = (gitBranch: IGitBranch, gitBranches: IGitBranch[]) =>
  gitBranches.map((branch: IGitBranch) =>
    gitBranch.name === branch.name ? {
      value: branch.name,
      label: branch.name,
      icon: "GitBranch"
    } : {
      value: branch.name,
      label: branch.name,
    }
  )
```